### PR TITLE
feat: add optional description field for services in config

### PR DESCRIPTION
**Pull request by zcstarr**

### CODE REVIEW

File: src/lib/service-runner-schema.json [hunk 1]
```diff
@@ -168,6 +168,9 @@
         },
         "name": {
           "type": "string"
+        },
+        "description": {
+          "type": "string"
```
Comment (shanejonas) [inline src/lib/service-runner-schema.json]:
```suggestion
          "type": "string",
          "description": "A verbose description of the application. GitHub Flavored Markdown syntax MAY be used for rich text representation."
```

Comment (shanejonas) [reply src/lib/service-runner-schema.json]: image

Comment (zcstarr) [reply src/lib/service-runner-schema.json]: Alot of offline convo and we settled on a different but similar solution in the PR update

**Pull request merged**